### PR TITLE
spring-props: - done;

### PR DESCRIPTION
**Pull request by jah701**

### CODE REVIEW

**Review by MichaelShevchenko — APPROVED** [review]
Everything seems fine to me. Good job

File: src/main/java/spring/intro/config/AppConfig.java [hunk 1]
```diff
@@ -3,25 +3,36 @@
...
 import spring.intro.model.User;
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "spring.intro.dao",
         "spring.intro.service"
 })
 public class AppConfig {
+    private final Environment env;
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/config/AppConfig.java]:
```suggestion
    private final Environment environment;
```

**Review by RomanDubovskyi — COMMENTED** [review]
change PR name, it's uninformative

**Comment by RomanDubovskyi** [comment]
this comments was not fixed

**Review by RomanDubovskyi — APPROVED** [review]
thsi PR name is uninformative, next time be more attentive with naming, it should reproduce functional that was implemented 
image

**Pull request merged**